### DISCUSSION TITLE
[epub.lua format q] Does ["bookmarks"] changed to ["annotations"]?

### DISCUSSION THREAD

**Discussion by kyxap** [koreader/koreader#11929] [Q&A] [answered]
Hi there,

I'm looking into some [plugin ](https://github.com/kyxap/koreader-calibre-plugin) which seems to stop syncing 'bookmarks' part and wandering if  ["bookmarks"] changed to ["annotations"] as part as book epub.lua metadata. 

Or does koreader has different formats per device? I do have clara wb which has `annotations` instead.
Anyway if I would appreciate if someone can point me to the code where the such format is configured and maybe can help to track when this was changed. 

Thank you?

**Comment by hius07** [comment, marked as answer]
The storage format changed in https://github.com/koreader/koreader/pull/11563.

**Comment by kyxap** [comment]
Thanks for quick response
